src/schema.types.ts

```diff
@@ -236,10 +249,50 @@ export function generateUseCaseOptionsInputType(
           values,
         }),
       },
+      ...(providerParameters && {
+        parameters: {
+          type: new GraphQLInputObjectType({
+            name: `${name}ProviderParameters`,
+            description: 'Provider-specific parameters',
+            fields: providerParameters,
+          }),
+        },
+      }),
     },
   });
 }
 
+export function generateUseCaseProviderParametersFields(
+  configuredProviders: string[],
+  allProviderSettings: ProviderSettingsRecord,
+): GraphQLInputFieldConfigMap | undefined {
+  // Set to prevent duplicated fields; it's okay if there are conflicting fields since we always expect a string
+  const parameterNames = new Set<string>();
+
+  // Generate a union of all parameters' names by all configured providers
+  for (const providerName of configuredProviders) {
+    const providerSettings = allProviderSettings[providerName];
+    const parameters = Object.keys(providerSettings.parameters);
+    parameters.forEach((parameterName) => parameterNames.add(parameterName));
+  }
+
+  if (parameterNames.size === 0) {
+    return undefined;
+  }
+
+  debug(
+    'generateUseCaseProviderParametersFields found provider parameters %o',
+    parameterNames,
+  );
+
+  const fields: GraphQLInputFieldConfigMap = {};
+  for (const parameterName of parameterNames.values()) {
+    fields[parameterName] = { type: GraphQLString };
```

Comment:
here should be parameter [description[(https://github.com/superfaceai/ast-js/blob/dev/src/interfaces/providerjson/providerjson.schema.json#L35-L37) set.

And as I mention in #10 also add supported providers to description as well.

That will improve user experience a lot.

<img width="356" alt="image" src="https://user-images.githubusercontent.com/959390/152523967-651b3976-967d-44fe-91cd-113d1acf2e21.png">